src/index.html

```diff
@@ -1,22 +1,474 @@
+<!-- #region head -->
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="widt64px !important;ice-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>Bang & Olufsen</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.scss"
     />
+
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+
+            <a
+              href="#"
+              class="top-bar__logo-link"
+            >
+              <img
+                src="images/bang&olufsen_logo.svg"
+                alt="Bang & Olufsen"
+                class="top-bar__logo"
+              />
+            </a>
+
+            <div class="top-bar__icons">
+              <a
+                href="#"
```

Comment:
ur link do nothing) u need to add valid logic for call
Check [Loom video](https://www.loom.com/share/7cd8c63cf3a942c2931f25d4bcded1d5)